MetriCam2/Camera.cs

```diff
@@ -1555,6 +1560,12 @@ public CameraImage CalcChannel(string channelName)
             //    return calcChannelCache[channelName];
             //}
 
+            if(imageCache.ContainsKey(channelName))
+            {
+                imageCache.TryGetValue(channelName, out CameraImage cachedImage);
```

Comment:
Is there a reason to use TryGetValue? I think `return imageCache[channelName]` is sufficient.